setup.py

```diff
@@ -44,6 +44,7 @@
         "nose2>=0.9.2",
```

Comment:
is it possible to read contents of requirments.txt so they appear only one time?

Reply:
Probably but don't want to touch it rn because it would cloud the PR more. It is already pretty big